docs/pages/includes/install-steps.mdx

```diff
@@ -0,0 +1,48 @@
+<Tabs>
```

Comment:
Refactored into an include so we can keep this consistent. We include these install steps in exactly two places:  (1) Getting Started page and (2) Installation page.

We refer to these steps in pretty much every other article, Guide, etc. I think doing this will help us improve the quality.